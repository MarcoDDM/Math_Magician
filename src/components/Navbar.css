nav  {
  list-style-type: none;
  display: flex;
}