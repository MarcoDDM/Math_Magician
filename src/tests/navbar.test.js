import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/navbar';
import '@testing-library/jest-dom/extend-expect';

describe('Test suite for the Navbar component', () => {
  let navbar;
  beforeEach(() => {
    render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );
    navbar = screen.getByText(/Home/i);
  });

  it('Should test if Navbar component is inside the document', () => {
    expect(navbar).toBeInTheDocument();
  });

  it('Should render a snapshot of the Navbar component', () => {
    expect(navbar).toMatchSnapshot();
  });
});
