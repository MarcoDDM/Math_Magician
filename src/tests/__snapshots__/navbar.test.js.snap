// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for the Navbar component Should render a snapshot of the Navbar component 1`] = `
<a
  class="nav-links"
  href="/"
>
  Home
</a>
`;
